index.html

```diff
@@ -113,6 +113,33 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card END ________  -->
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Himanshu Dhir</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="https://twitter.com/home" target="_blank">Himanshu54276066</a>
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/himanshurd" target="_blank">himanshurd</a>
+          </p>
+          <p class="about">about you</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://www.linkedin.com/feed/?trk=guest_homepage-basic_nav-header-signin" target="_blank" title="First Resource">This is my Linkdin's Account</a>
+              </li>
+              <li>
+                <a href="" target="_blank" title="Second Resource">My hobby is to play cricket. I love to learn more about AI using Python</a>
```

Comment:
Oh no! Looks like there's no target for this link tag. (Your `href` is empty in your `<a>` tag.)